tests/python/unittest/test_runtime_module_based_interface.py

```diff
@@ -674,6 +675,19 @@ def make_module(mod):
     module_main.get_function("func_b", query_imports=True)
 
 
+def test_num_threads():
+    reported = tvm.runtime.num_threads()
+    env_threads = os.getenv("TVM_NUM_THREADS")
+    omp_env_threads = os.getenv("OMP_NUM_THREADS")
+    if env_threads is not None:
+        assert reported == env_threads
+    elif omp_env_threads is not None:
+        assert reported == omp_env_threads
+    else:
+        hardware_threads = os.cpu_count() / 2
+        assert reported == hardware_threads
```

Comment:
I think this is only true on x64 https://github.com/apache/tvm/blob/6ad7638be249fef0b2ce3f1112dd857e850949f1/src/runtime/threading_backend.cc#L304-L305

Reply:
Changed the test to check if the reported number is `os.cpu_count()` or `os.cpu_count()/2`. That sound reasonable?